Content.Shared/Humanoid/SkinColor.cs

```diff
@@ -131,7 +132,7 @@ public static Color TintedHues(Color color)
     public static bool VerifyTintedHues(Color color)
     {
         // tinted hues just ensures saturation is always .1, or 10% saturation at all times
-        return Color.ToHsv(color).Y != .1f;
+        return Color.ToHsl(color).Y <= .1f && Color.ToHsl(color).Y >= .85f;
```

Comment:
can we put 0.1f and 0.85f into consts? (those numbers are used in `TintedHues` method above too)